crt_portal/cts_forms/forms.py

```diff
@@ -1263,35 +1263,36 @@ class PrintActions(Form):
 
 
 class BulkActions(Form, ActivityStreamUpdater):
```

Comment:
Not related to the functionality here but I'd love to rename this for `BulkActionsForm` and help nudge the entire codebase towards more idiomatic Django class names.

Reply:
I agree. Renamed in this PR, but I would also suggest a follow up PR right after this one since we have other form names: `Filters`, `ResponseActions`, `ComplaintActions`, and `PrintActions` should be renamed accordingly.